src/main/resources/application.properties

```diff
@@ -13,4 +13,5 @@ management.endpoint.health.enabled=true
 management.endpoint.prometheus.enabled=true
 management.endpoints.web.exposure.include=prometheus,health
 
-featureToggle.useCryptoService = false
\ No newline at end of file
+featureToggle.useCryptoService = false
```

Comment:
Kanskje bruke "useCryptoServiceForEncrypt", siden vi også skal ha en toggle for "decrypt"?